distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -387,16 +387,18 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
     }
 
     protected Option[] configurePaxExam() {
-        return options(logLevel(LogLevelOption.LogLevel.INFO),
-                useOwnExamBundlesStartLevel(100),
+        return options(logLevel(LogLevelOption.LogLevel.INFO), useOwnExamBundlesStartLevel(100),
                 // increase timeout for CI environment
-                systemTimeout(TimeUnit.MINUTES.toMillis(10)),
-                when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-                cleanCaches(true));
+                systemTimeout(TimeUnit.MINUTES.toMillis(10)), when(Boolean.getBoolean(
+                        "keepRuntimeFolder")).useOptions(keepRuntimeFolder()), cleanCaches(true));
     }
 
     protected Option[] configureAdditionalBundles() {
         return options(junitBundles(),
+                wrappedBundle(mavenBundle("ddf.catalog.registry",
```

Comment:
This was added because our tests directly interact with the framework so we are creating Metacard objects and such. It is this way because currently we do not have an external way of submitting request via csw with destination parameters.